pandas/core/indexes/datetimes.py

```diff
@@ -1707,8 +1707,69 @@ def freq(self, value):
     weekofyear = _field_accessor('weekofyear', 'woy',
                                  "The week ordinal of the year")
     week = weekofyear
-    dayofweek = _field_accessor('dayofweek', 'dow',
-                                "The day of the week with Monday=0, Sunday=6")
+    _dayofweek_doc = """
+    The day of the week with Monday=0, Sunday=6.
+
+    Return the day of the week. It is assumed the week starts on
+    Monday, which is denoted by 0 and ends on Sunday which is denoted
+    by 6. This method is available on both Series with datetime
+    values or DatetimeIndex
+
+    See Also
+    --------
+    pandas.Series.dt.dayofweek : Identical method.
+    pandas.Series.dt.weekday : Identical method.
+
+    Returns
+    -------
+    Series
+        Containing integers indicating the week number.
+
+    ndarray of integers indicating the day of the week
+
+    Examples
+    --------
+    >>> dates = pd.date_range("2016-12-31", "2017-01-08", freq="D")
+    >>> s = pd.Series(dates)
+    >>> day_name = s.dt.strftime("%A")
```

Comment:
we have a method ``day_name`` for this

Reply:
did not know this. i was confused because i saw `weekday_name` but this was depricated. i've changed the code and added `day_name` to the "see also" too.

Reply:
change the code here as well and simply use ``day_name``